src/worker/extensions/bouncer/index.js

```diff
@@ -9,14 +9,14 @@ module.exports.init = async function init(hooks, app) {
         }
 
         upstream.forEachClient(client => {
-            if (client.state.caps.includes('BOUNCER')) {
+            if (client.state.caps.has('BOUNCER')) {
```

Comment:
hmm, is this supposed to be 'BOUNCER'? caps are always lowercase, maybe that should be changed here as well.